Languages/English/Keyed/Keys.xml

```diff
@@ -1,29 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <LanguageData>
 
-	<koisama.Numbers.Title>Numbers overview</koisama.Numbers.Title>
-	<koisama.Numbers.Count>Row Count</koisama.Numbers.Count>
-	<koisama.Numbers.ClickToToggle>Click to select pawn type.</koisama.Numbers.ClickToToggle>
+	<koisama.Numbers.Title>Numbers!</koisama.Numbers.Title>
```

Comment:
I share your excitement for Numbers! yet still prefer the more toned down wording.